### ISSUE TITLE
Diffrences from PyTorch implementation

### ISSUE THREAD

**Issue by fanpengustc** [goldkim92/StarGAN-tensorflow#4]
Generator has no ResidualBlock
Why did you add Real target image ? Just label is enough!!!
The training is so confused, I think it is different from the PyTorch implementation
